src/show/components/DriverCard/index.jsx

```diff
@@ -1,4 +1,4 @@
-const DriverCard = ({ car, firstDriver, secondDriver }) => {
+const DriverCard = ({ car_url, firstDriver, secondDriver }) => {
```

Comment:
Prop should be `carUrl`, even though the attribute is `car_url` (comes from rails).